examples/python2.7/BUILD

```diff
@@ -0,0 +1,18 @@
+load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build")
+load("@subpar//:subpar.bzl", "par_binary")
+
+par_binary(
+    name = "hello_py",
+    srcs = ["hello.py"],
+    main = "hello.py",
+)
+
+docker_build(
+    name = "hello",
+    base = "//python2.7:python27",
+    cmd = [
+        "/hello_py.par",
+        "/etc",
```

Comment:
Can you document why you are passing `/etc` here?  It is weird and an abundance of documentation wouldn't hurt.